h/services/annotation_sync.py

```diff
@@ -167,10 +174,11 @@ def get(self, jobs: list[Job]) -> dict:
 
         for hit in hits:
             updated = hit["_source"].get("updated")
-            updated = isoparse(updated).replace(tzinfo=None) if updated else None
-            hit["_source"]["updated"] = updated
+            if updated:
+                updated = isoparse(updated).replace(tzinfo=None)
+                hit["_source"]["updated"] = updated
```

Comment:
Small fix to not insert `"updated": None` into `source` if `source` didn't already contain an `"updated"` (which happens when annotations have been deleted from Elasticsearch, the `"source"` for deleted annotations in Elasticsearch is `{deleted: true}`.

This is necessary for the `if` below, so that `hit["_source"]` remains falsey for deleted annotations.